src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -343,6 +345,70 @@ def _test_firewall_mgmt(self, resource_group, server, database_engine):
         self.cmd('{} server firewall-rule list -g {} --server {}'
                  .format(database_engine, resource_group, server), checks=[NoneCheck()])
 
+    def _test_vnet_firewall_mgmt(self, resource_group, server, database_engine):
+        vnet_firewall_rule_1 = 'vnet_rule1'
+        vnet_firewall_rule_2 = 'vnet_rule2'
+        location = 'koreasouth'
+        vnet_name = 'clitestvnet'
+        ignore_missing_endpoint = 'true'
+        address_prefix = '10.0.0.0/16'
+
+        subnet_name_1 = 'clitestsubnet1'
+        subnet_prefix_1 = '10.0.0.0/24'
+
+        subnet_name_2 = 'clitestsubnet2'
+        subnet_prefix_2 = '10.0.1.0/24'
+
+        # pre create the dependent resources here
+        # create vnet and subnet
+        execute(self.cli_ctx, 'az network vnet create -n {} -g {} -l {} '
```

Comment:
These should be written using kwargs notation for better readability. See the documentation for a examples:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md#sample-6-random-name-and-name-mapping